programs/fileio.c

```diff
@@ -973,10 +1041,19 @@ static cRess_t FIO_createCResources(FIO_prefs_t* const prefs,
     /* need to update memLimit before calling createDictBuffer
      * because of memLimit check inside it */
     if (prefs->patchFromMode) {
+        U64 const dictSize = UTIL_getFileSize(dictFileName);
```

Comment:
We should reorganize this so that we aren't stat-ing a file multiple times for patch-from.